lib/anyplayer/selector.rb

```diff
@@ -71,21 +69,21 @@ def player_load(player)
 
     def player_on_platform?(player)
       result = player.platforms.include?(platform)
-      log "  not on platform" if !result
+      log '  not on platform' unless result
       result
     end
 
     def player_class(player)
-      camelized = player.to_s.split(/_/).map{ |word| word.capitalize }.join
-      Anyplayer::const_get(camelized)
+      camelized = player.to_s.split(/_/).map(&:capitalize).join
+      Anyplayer.const_get(camelized)
     end
 
     def player_launched?(player)
       seconds = 5
       begin
-        Timeout::timeout(seconds) do
+        Timeout.timeout(seconds) do
           launched = player.launched?
-          log launched ? "  launched" : "  not launched"
+          log launched ? '  launched' : '  not launched'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.